solana/programs/example-native-token-transfers/src/instructions/redeem.rs

```diff
@@ -22,6 +22,10 @@ pub struct Redeem<'info> {
     pub payer: Signer<'info>,
 
     // NOTE: this works when the contract is paused
+    #[account(
+        constraint = config.next_transceiver_id != 0 @ NTTError::NoRegisteredTransceivers,
+        constraint = config.enabled_transceivers.is_empty() @ NTTError::NoRegisteredTransceivers,
```

Comment:
This adds parity with the EVM implementation:
https://github.com/wormhole-foundation/example-native-token-transfers/blob/0d764bf7ffbbf81b4d51d1bfd5e51929e7fb0460/evm/src/NttManager/ManagerBase.sol#L260